app/assets/stylesheets/sufia/_fixedsticky.scss

```diff
@@ -0,0 +1,28 @@
+.fixedsticky {
```

Comment:
fixedsticky dosen't provide/maintain this, but they show this as an example of how to style it.